components/raftstore/src/store/util.rs

```diff
@@ -785,7 +785,7 @@ impl<'a> ChangePeerI for &'a ChangePeerRequest {
         let mut cc = eraftpb::ConfChange::default();
         cc.set_change_type(self.get_change_type());
         cc.set_node_id(self.get_peer().get_id());
-        cc.set_context(ctx);
+        cc.set_context(ctx.into());
```

Comment:
How about making `to_confchange` accept `Bytes`?

Reply:
Then `to_confchange` will be hard to implement for `prost-codec`.

Considering `ctx` is not copied in the caller of `to_confchange`(components/raftstore/src/store/peer.rs:3180), let's keep it.